.github/workflows/publish-release.yml

```diff
@@ -12,17 +12,17 @@ jobs:
       id-token: write
     steps:
       - uses: actions/checkout@v5
+      - uses: pnpm/action-setup@v4
+        with:
+          package_json_file: package.json
       - uses: actions/setup-node@v5
         with:
           node-version-file: package.json
-          cache: "npm"
```

Comment:
Can't we do caching with pnpm too? Could you check the readme for https://github.com/pnpm/action-setup to see their example with caching?